### PR TITLE
Add SummaryList component

### PR DESCRIPTION
**Pull request by rafenden**
image

### CODE REVIEW

File: src/summary-list/__tests__/SummaryList.test.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
```
Comment (paulgain) [inline src/summary-list/__tests__/SummaryList.test.jsx]: Superfluous space.

File: src/summary-list/__tests__/SummaryList.test.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import Link from '@govuk-react/link'
+import SummaryList from '../SummaryList'
```
Comment (paulgain) [inline src/summary-list/__tests__/SummaryList.test.jsx]: Space after `import Link from '@govuk-react/link'`

File: src/summary-list/__tests__/SummaryList.test.jsx (hunk 1 above)
Comment (rafenden) [reply src/summary-list/__tests__/SummaryList.test.jsx]: Fixed

File: src/summary-list/__tests__/SummaryList.test.jsx (hunk 2 above)
Comment (rafenden) [reply src/summary-list/__tests__/SummaryList.test.jsx]: Fixed

**Pull request merged**